tests/test_qp.py

```diff
@@ -25,12 +25,12 @@ class QPTest(PyverbsAPITestCase):
     Test various functionalities of the QP class.
     """
 
-    def create_qp(self, creator, qp_init_attr, is_ex, with_attr):
+    def create_qp(self, creator, qp_init_attr, is_ex, with_attr, port_num):
```

Comment:
Is there a reason why you did not assign 1 as the default port_num value?

Reply:
I'll fix it in v2.